apps/mobile/src/app/settings/wallet/configure/[wallet]/[account]/index.tsx

```diff
@@ -93,7 +93,7 @@ function ConfigureAccount({ fingerprint, accountIndex, account }: ConfigureAccou
                 onPress={() => {
                   router.navigate({
                     pathname: AppRoutes.Account,
-                    params: { accountId: account.id },
+                    params: { account: account.id, accountId: account.id },
```

Comment:
At some point we should use the typed value of `account` rather than the `accountId`

Reply:
I don't understand this change but I guess it's needed? 

Don't we pass the same thing twice here? 
```
account: account.id, accountId: account.id
```